src/Jackalope/Transport/DoctrineDBAL/CachedClient.php

```diff
@@ -339,27 +402,11 @@ public function moveNodeImmediately($srcAbsPath, $dstAbsPath)
     /**
      * {@inheritDoc}
      */
-    public function reorderNodes($absPath, $reorders)
```

Comment:
also this change is actually part of #280 

to review this maybe better to look at https://github.com/jackalope/jackalope-doctrine-dbal/compare/cache_queries...cache_queries_node_types